/soundfont.dls": No such file or directory / sonivox 3.6.15 fails to build on debian

Here is the output:

```
Command: dpkg-buildpackage --sanitize-env -us -uc -rfakeroot
dpkg-buildpackage: info: source package sonivox
dpkg-buildpackage: info: source version 3.6.15-1
dpkg-buildpackage: info: source distribution UNRELEASED
dpkg-buildpackage: info: source changed by Dennis Braun <snd@debian.org>
 dpkg-source --before-build .
dpkg-buildpackage: info: host architecture amd64
 debian/rules clean
dh clean
   dh_clean
 dpkg-source -b .
dpkg-source: info: using source format '3.0 (quilt)'
dpkg-source: info: building sonivox using existing ./sonivox_3.6.15.orig.tar.gz
dpkg-source: info: building sonivox in sonivox_3.6.15-1.debian.tar.xz
dpkg-source: info: building sonivox in sonivox_3.6.15-1.dsc
 debian/rules binary
dh binary
   dh_update_autotools_config
   dh_autoreconf
   debian/rules override_dh_auto_configure
make[1]: Entering directory '/<<PKGBUILDDIR>>'
dh_auto_configure
	cd obj-x86_64-linux-gnu && DEB_PYTHON_INSTALL_LAYOUT=deb PKG_CONFIG=/usr/bin/pkg-config cmake -DCMAKE_INSTALL_PREFIX=/usr -DCMAKE_BUILD_TYPE=None -DCMAKE_INSTALL_SYSCONFDIR=/etc -DCMAKE_INSTALL_LOCALSTATEDIR=/var -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON -DFETCHCONTENT_FULLY_DISCONNECTED=ON -DCMAKE_INSTALL_RUNSTATEDIR=/run -DCMAKE_SKIP_INSTALL_ALL_DEPENDENCY=ON "-GUnix Makefiles" -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_INSTALL_LIBDIR=lib/x86_64-linux-gnu ..
-- The C compiler identification is GNU 14.2.0
-- The CXX compiler identification is GNU 14.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE
-- Found GTest v1.15.0
-- Found Math library: /usr/lib/x86_64-linux-gnu/libm.so
CMake Error at CMakeLists.txt:311 (file):
  file DOWNLOAD cannot open file for write.


CMake Error at CMakeLists.txt:314 (file):
  file SIZE requested of path that is not readable:

    /soundfont.dls


CMake Error at CMakeLists.txt:315 (file):
  file MD5 failed to read file "/soundfont.dls": No such file or directory


-- DLS file /soundfont.dls size: 
-- DLS file /soundfont.dls MD5 hash: 
-- Configuring incomplete, errors occurred!
```

Please consider adding a dls file to the repo rather than download it, because the debian policy does not allow to download files while building.

--- a/test/SonivoxTestEnvironment.h
+++ b/test/SonivoxTestEnvironment.h
@@ -28,11 +28,16 @@ class SonivoxTestEnvironment : public::testing::Environment {
 	SonivoxTestEnvironment() : deleteOutput(true) { 
 		if (getenv("TEMP") != nullptr) {
             tmp = getenv("TEMP");
-            if (tmp.at(tmp.length() - 1) != '/') {
-                tmp.push_back('/');
-            }
-            snprintf(OUTPUT_FILE, sizeof(OUTPUT_FILE), "%s/output_midi.pcm", getenv("TEMP"));
+        } else if (getenv("XDG_RUNTIME_DIR") != nullptr) {
+            tmp = getenv("XDG_RUNTIME_DIR");
+        } else {
+            tmp = "/tmp/";
         }
+        if (tmp.at(tmp.length() - 1) != '/') {
+            tmp.push_back('/');
+        }
+        snprintf(OUTPUT_FILE, sizeof(OUTPUT_FILE), "%s/output_midi.pcm", getenv("XDG_RUNTIME_DIR"));
+
         if (getenv("TEST_RESOURCES") != nullptr) {
             res = getenv("TEST_RESOURCES");
         }

--- a/test/SonivoxTestEnvironment.h
+++ b/test/SonivoxTestEnvironment.h
@@ -36,7 +36,7 @@ class SonivoxTestEnvironment : public::testing::Environment {
         if (tmp.at(tmp.length() - 1) != '/') {
             tmp.push_back('/');
         }
-        snprintf(OUTPUT_FILE, sizeof(OUTPUT_FILE), "%s/output_midi.pcm", getenv("XDG_RUNTIME_DIR"));
+        snprintf(OUTPUT_FILE, sizeof(OUTPUT_FILE), "%soutput_midi.pcm", tmp.c_str());
 
         if (getenv("TEST_RESOURCES") != nullptr) {
             res = getenv("TEST_RESOURCES");

